test/bench/avro-insert/.gitignore

```diff
@@ -0,0 +1 @@
+data
```

Comment:
I have been wondering what this ignores. I blindly copied it before but I never actually saw a `data` directory being created.

Reply:
Oh, this is likely vestigial from chbench then. I asked a similar thing on #6669 and it sounds like I may have accidentally copied this file as well.

I'll remove this `.gitignore`.